apps/blockchain/starknet/src/tests/bridge_t.cairo

```diff
@@ -1332,12 +1336,12 @@ mod tests {
         assert_eq!(erc721.owner_of(0), OWNER_L2, "Token should be transferred to OWNER_L2");
 
         // Verify escrow is cleared
-        let is_escrowed = load(
+        let escrowed = load(
             bridge_address,
             map_entry_address(selector!("escrow"), array![erc721b_address.into(), 0, 0].span()),
             1
-        )
-            .is_zero();
+        );
+        let is_escrowed = escrowed.len() == 0 && (*escrowed[0]).is_zero();
```

Comment:
This test is wrong:
- if address in escrow map is zero it means that token is not in escrow.
- `escrowed.len() == 0` will always be false since we expect one value from the storage.

Please correct with this:
```cairo
let is_escrowed = escrowed.len() == 1 && (*escrowed[0]).is_non_zero();
```

Reply:
@ptisserand , i have made the changes as requested